dex/models/_projects/jelly_swap/jelly_swap_sei_trades.sql

```diff
@@ -0,0 +1,38 @@
+{{ config(
+        schema = 'jelly_swap',
+        alias = 'trades',
+        materialized = 'view',
+        post_hook='{{ expose_spells(blockchains = \'["sei"]\',
+                                      spell_type = "project", 
+                                      spell_name = "jelly_swap", 
+                                      contributors = \'["viniabussafi"]\') }}'
```

Comment:
```suggestion
        materialized = 'view'
```

Reply:
we're leaning towards displaying less in the data explorer up front. open to adding, if there is strong case, but most users will query `dex.trades` and filter on projects rather than use these views directly